validation/standard/standard-versions.toml

```diff
@@ -7,8 +7,4 @@ optimism_mintable_erc20_factory = "1.9.0"
 optimism_portal = "3.10.0"
 system_config = "2.2.0"
 protocol_versions = "1.0.0"
-anchor_state_registry = "1.0.0"
-delayed_weth = "1.0.0"
-dispute_game_factory = "1.0.0"
-fault_dispute_game = "1.2.0"
```

Comment:
I think we can keep these?

Reply:
There was an error with anchor state registry mismatch for OP sepolia. I see this [thread](https://oplabs-pbc.slack.com/archives/C073P71SGP2/p1723932925971519?thread_ts=1723932048.355599&cid=C073P71SGP2) where you've mentioned the right way to address this. I've added an exclusion for Sepolia in `Standard_Contract_Versions`.